library/std/src/sys_common/condvar/check.rs

```diff
@@ -9,7 +10,7 @@ pub trait CondvarCheck {
 
 /// For boxed mutexes, a `Condvar` will check it's only ever used with the same
 /// mutex, based on its (stable) address.
```

Comment:
is this still a good comment? maybe "boxed mutexes" should link to `LazyBox`?

Reply:
They're still boxed. That they're lazily allocated isn't really relevant for this comment.